src/Paket.Core/Dependencies/PackageResolver.fs

```diff
@@ -877,8 +882,18 @@ let Resolve (getVersionsF, getPackageDetailsF, groupName:GroupName, globalStrate
                                            This will result in an endless loop.%sCurrent Requirement: %A%sRequirements: %A"
                                                 Environment.NewLine currentRequirement Environment.NewLine nextStep.OpenRequirements
                             step (Step((currentConflict,nextStep,currentRequirement), (currentConflict,currentStep,currentRequirement,compatibleVersions,flags)::priorConflictSteps)) stackpack currentConflict.VersionsToExplore flags
-                        else
-                            step (Inner ((currentConflict,currentStep,currentRequirement), priorConflictSteps)) stackpack compatibleVersions flags
+                        | firstError :: _ ->
+                            let conflict =
+                                firstError
+                                |> Seq.map (fun (name,vr,_) -> 
+                                        { currentRequirement with
+                                              Name = name
+                                              VersionRequirement = vr
+                                              Parent = PackageRequirementSource.Package(currentRequirement.Name, exploredPackage.Version, exploredPackage.Source)
+                                              Graph = currentRequirement :: currentRequirement.Graph })
+                                |> Seq.fold (fun conflict r -> { conflict with Conflicts = Set.add r currentConflict.Conflicts }) currentConflict
```

Comment:
I actually wanted to do this. But I couldn't figure out all this conflict stuff, sorry :/